languages/cp-plgn-drctry.pot

```diff
@@ -0,0 +1,75 @@
+# Copyright (C) 2022 Cp_Plgn_Drctry
```

Comment:
Given that the up next version is 1.3.0 I think we can exclude this entire content. 
We can generate the .po just before releasing 1.3 (there are many string changes in there)

Of course we can also include it now and then just regenerate it later.

Reply:
Of course. The important thing is if the build makes the work easy. We can always regenerate the file when ready.